CHANGELOG

```diff
@@ -1,6 +1,18 @@
 CHANGES
 =======
 
+v1.1
+----
+
+* Support removal of deprecated behaviour and convenience imports in Jinja2 3.1+
```

Comment:
Is it worth adding a note here that Python 2 and Django 2 support is officially dropped?

Reply:
Ah yeah, I see your point. Tho setup.py at least reflects the official support, it'd be nice to add to the changelog too